packages/core/src/__integrationtests__/Balance.spec.ts

```diff
@@ -80,7 +80,7 @@ describe('when there is a dev chain with a faucet', () => {
     expect(balanceBefore.sub(balanceAfter).gt(MIN_TRANSACTION)).toBeTruthy()
     expect(balanceIdent.toNumber()).toBe(MIN_TRANSACTION.toNumber())
     expect(funny).toBeCalled()
-  }, 30_000)
+  }, 60_000)
```

Comment:
Why more?

Reply:
Tests will take more time with the parachain. Blocktime is 12s not 6s. Also the blocktime is not constant. It takes longer sometimes.